src/components/Finishers-Components/Finisher-Group.vue

```diff
@@ -178,87 +178,9 @@ export default {
     color: #222222;
 }
 
-.finisher-filter{
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    width: 25%;
-    height: fit-content;
-    background-color: white;
-    padding: 25px;
-    box-sizing: border-box;
-    box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.1);
-}
-
-.finisher-filter h5{
-    font-family: "Open Sans", sans-serif;
```

Comment:
Dont forget to remove open sans font in css classes since 'Google Sans' is specified globally. Thanks!